selfdrive/car/toyota/carcontroller.py

```diff
@@ -63,7 +63,7 @@ def process_hud_alert(hud_alert, audible_alert):
     sound1 = 1
   elif audible_alert in ['beepSingle', 'chimeSingle', 'chimeDouble']:
     # TODO: find a way to send single chimes
-    sound2 = 1
+    sound2 = 0 #Changed by P Lee to disable disengage chime
```

Comment:
@zeeexsixare : can you remove the changes to `carcontroller.py`? Also, can you update the README by lowering the min accel speed to zero?

Reply:
This is done now.